server/routes/api.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express')
+const { read } = require('fs')
+const router = express.Router()
+const moment = require('moment')
+const mongoose = require('mongoose')
+
+const Expense = require('../model/Expenses')
+const Schema = mongoose.Schema
+
+//Getting Expenses
+router.get('/expenses',function(request,response){
+    Expense.find({}).sort({"date":1}).exec(function(err,expenses){
+        response.send(expenses)
+    })
+})
+
+//Add New Expenses
+router.post('/expense',function(request,response){
+    let postObj = {
+        name : request.body.name,
+        amount:request.body.amount,
+        date:(request.body.date)?moment(request.body.date).format("LLLL"):moment(new Date()).format("LLLL"),
+        group:request.body.group
+    }
```

Comment:
you could just do - 
`request.body.date = moment(request.body.date ? request.body.date : new Date()).format("LLLL")`